manifests/1/14/00/14.0.30035.0/14.00.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: 14.00
+PackageVersion: 14.0.30035.0
+PackageLocale: en-US
+Publisher: Microsoft Platform Extensions

```

Comment:
```suggestion
Publisher: Microsoft Corporation
```

Reply:
cc @Trenly: It's supposed to be `Microsoft Platform Extensions`.

```
Name              : Microsoft Visual C++ 2015 UWP Desktop Runtime Package
Publisher         : Microsoft Platform Extensions
Version           : 14.0.30035.0
PackageFamilyName : Microsoft.VCLibs.140.00.UWPDesktop_8wekyb3d8bbwe
Uninstall         : winget uninstall "Microsoft.VCLibs.140.00.UWPDesktop_8wekyb3d8bbwe"
```